test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -6474,6 +6474,9 @@ def test_embedding_bag(self):
         offset = torch.tensor([0, 2, 5, 6])
         self.run_test(model, (input, offset))
 
+    @disableScriptTest()  # error in propagate as assign input shape
+    @skipIfUnsupportedMinOpsetVersion(11)
+    def test_embedding_bag_with_offset(self):
```

Comment:
Similar to above, offset becomes dynamic, hence requires opset 11.
Next PR in the stack reverts this effect.